docs/core/additional-tools/uninstall-tool.md

```diff
@@ -2,45 +2,46 @@
 title: Uninstall Tool
 description: An overview of the .NET Uninstall Tool, a guided tool that enables the controlled clean-up of .NET SDKs and runtimes.
 author: sfoslund
+ms.custom: devdivchpfy22
 ms.date: 07/28/2022
 ---
-# .NET Uninstall Tool
+# .NET uninstall tool
```

Comment:
Should this be lowercase everywhere considering it's the property name of the product being documented here? .NET Uninstall Tool is the name of the product: https://devblogs.microsoft.com/dotnet/announcing-the-net-core-uninstall-tool-1-0/